test/external-modules/esql-heap-attack/src/javaRestTest/java/org/elasticsearch/xpack/esql/heap_attack/HeapAttackIT.java

```diff
@@ -624,6 +628,53 @@ private Response fetchMvLongs() throws IOException {
         return query(query.toString(), "columns");
     }
 
+    public void testLookupExplosion() throws IOException {
+        int sensorDataCount = 1000;
+        int lookupEntries = 100;
+        Map<?, ?> map = responseAsMap(lookupExplosion(sensorDataCount, lookupEntries));
+        assertMap(map, matchesMap().extraOk().entry("values", List.of(List.of(sensorDataCount * lookupEntries))));
+    }
```

Comment:
Having the next test, do we need this one? Maybe move the assertMap to the next one and remove this?
Same for enrich

Reply:
We we'll just break in the other path and never get any results though. I could move the `assertMap` into there and rely on it throwing, but I think this is a little clearer way to do it.